examples/account-management.ipynb

```diff
@@ -0,0 +1,264 @@
+     "name": "#%%\n"
+    }
+   }
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "outputs": [],
+   "source": [
+    "# create and save legacy account\n",
+    "IBMRuntimeService.save_account(\n",
+    "    name=\"legacy-account-1\",\n",
+    "    account=dict(type=\"legacy\", token=\"123\"),\n",
+    ")"
+   ],
+   "metadata": {
+    "collapsed": false,
+    "pycharm": {
+     "name": "#%%\n"
+    }
+   }
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "outputs": [],
+   "source": [
+    "# overwrite legacy account and add default HGP information\n",
+    "IBMRuntimeService.save_account(\n",
+    "    name=\"legacy-account-1\",\n",
+    "    account=dict(type=\"legacy\", token=\"123\", hub=\"ibm-q\", group=\"open\", project=\"main\"),\n",
+    "    overwrite=True,\n",
+    ")"
+   ],
+   "metadata": {
+    "collapsed": false,
+    "pycharm": {
+     "name": "#%%\n"
+    }
+   }
+  },
+  {
+   "cell_type": "markdown",
+   "source": [
+    "### Delete account"
+   ],
+   "metadata": {
+    "collapsed": false
+   }
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "outputs": [],
+   "source": [
+    "IBMRuntimeService.delete_account(name=\"cloud-account-1\")"
+   ],
+   "metadata": {
+    "collapsed": false,
+    "pycharm": {
+     "name": "#%%\n"
+    }
+   }
+  },
+  {
+   "cell_type": "markdown",
+   "source": [
+    "### List accounts"
+   ],
+   "metadata": {
+    "collapsed": false
+   }
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "outputs": [],
+   "source": [
+    "IBMRuntimeService.list_accounts()"
+   ],
+   "metadata": {
+    "collapsed": false,
+    "pycharm": {
+     "name": "#%%\n"
+    }
+   }
+  },
+  {
+   "cell_type": "markdown",
+   "source": [
+    "## Usage in IBMRuntimeService"
+   ],
+   "metadata": {
+    "collapsed": false
+   }
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "outputs": [],
+   "source": [
+    "# below calls are equivalent (i.e. if omitted the default account is used)\n",
+    "service1 = IBMRuntimeService()\n",
+    "service2 = IBMRuntimeService(account=IBMRuntimeService.load_default_account())"
+   ],
+   "metadata": {
+    "collapsed": false,
+    "pycharm": {
+     "name": "#%%\n"
+    }
+   }
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "outputs": [],
+   "source": [
+    "# below calls are equivalent\n",
+    "service3 = IBMRuntimeService(account_name=\"cloud-account\")\n",
+    "service4 = IBMRuntimeService(\n",
+    "    account=IBMRuntimeService.load_account(name=\"cloud-account\")\n",
+    ")"
+   ],
+   "metadata": {
+    "collapsed": false,
+    "pycharm": {
+     "name": "#%%\n"
+    }
+   }
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "outputs": [],
+   "source": [
+    "# users who don't need account management can still pass in account info manually\n",
+    "service5 = IBMRuntimeService(account=dict(type=\"legacy\", token=\"123\"))"
```

Comment:
token and locator should be direct parameters here, we don’t even need type here I guess